roles/matrix-client-cinny/tasks/validate_config.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Fail if required Cinny settings not defined
+  fail:
+    msg: >
+      You need to define a required configuration setting (`{{ item }}`) to use Cinny.
+  when: "(vars[item] == '' or vars[item] is none) and matrix_client_cinny_container_image_self_build|bool"
```

Comment:
Why do we have `and matrix_client_cinny_container_image_self_build|bool` here?

Reply:
I used `matrix-client-hydrogen` as template and seems that condition was added because hydrogen can be only self-built, but that's not true for cinny, so removed it